.github/workflows/continuous-integration.yml

```diff
@@ -24,7 +21,7 @@ jobs:
         node-version: [12, 14, 16, 17]
         include:
           - os: macos-latest
-            node-version: 14 # LTS
+            node-version: 16 # LTS
```

Comment:
This is already done by https://github.com/loopbackio/loopback-next/pull/8088

Reply:
I do rebase